localization/ndt_scan_matcher/include/ndt_scan_matcher/map_module.hpp

```diff
@@ -0,0 +1,50 @@
+// Copyright 2022 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NDT_SCAN_MATCHER__MAP_MODULE_HPP_
+#define NDT_SCAN_MATCHER__MAP_MODULE_HPP_
+
+#include <rclcpp/rclcpp.hpp>
+
+#include <sensor_msgs/msg/point_cloud2.hpp>
+
+#include <fmt/format.h>
```

Comment:
`fmt/format.h` is not used in the map_module. So you can remove this line.

Reply:
Thanks! Done: https://github.com/autowarefoundation/autoware.universe/pull/2249/commits/d7e4c7a6de24c68113e0c7b5a8d8b82601c29373